api/Pipfile

```diff
@@ -34,15 +34,14 @@ drf-spectacular = "*"
 elasticsearch = "==8.8.2"
 elasticsearch-dsl = "~=8.9"
 future = "~=0.18"
-gunicorn = "~=21.2"
 limit = "~=0.2"
 Pillow = "~=10.0.1"
 psycopg2 = "~=2.9"
 python-decouple = "~=3.8"
 python-xmp-toolkit = "~=2.0"
 sentry-sdk = "~=1.30"
 django-split-settings = "*"
-uvloop = "~=0.17"
```

Comment:
Is this removing `uvloop`?

Reply:
[`uvicorn[standard]` already includes it along with other significant improvements to the stock asyncio interface and sets the asyncio loop policy for it](https://www.uvicorn.org/#quickstart), so we don't need to add an explicit dependency (which could lead to weird version drift between uvicorn and uvloop if we don't update them together). We do still need to keep the uvloop policy setting in `manage.py`, though, because we don't run that with uvicorn, just regular old Python.